src/program.js

```diff
@@ -569,29 +569,31 @@ Example: $0 --help run.
           describe:
             'Use a proxy to access the signing API. ' +
             'Example: https://yourproxy:6000 ',
-          demandOption: false,
```

Comment:
are these removed `demandOption: false` related to this set of changes?

Reply:
k, then let's omit it from the new one and add back the ones that are technically the unrelated change for this PR, we can consider doing that if that is the yargs default now, but I'd prefer to be doing that in a separate PR.